src/extension.ts

```diff
@@ -72,28 +72,36 @@ export function activate(context: vscode.ExtensionContext) {
 
   status.tooltip = 'Path to tag'
 
-  vscode.window.onDidChangeTextEditorSelection(() => {
-    const editor = vscode.window.activeTextEditor
-
-    if (!config.isEnabled || !editor) {
-      return
-    }
-
-    const tagsList = parseTags(editor.document.getText())
-    const position = editor.document.offsetAt(editor.selection.active)
-
-    // Highlight matching tag
-    const match = findMatchingTag(tagsList, position)
-
-    // Tag breadcrumbs
-    if (config.showPath) {
-      updateTagStatusBarItem(status, tagsList, position)
-    }
-
-    if (match && (match.opening !== match.closing || config.highlightSelfClosing)) {
-      tagStyler.decoratePair(match, editor)
-    } else {
-      tagStyler.clearDecorations()
-    }
-  })
+  context.subscriptions.push(
+    vscode.window.onDidChangeTextEditorSelection(() => {
+      const editor = vscode.window.activeTextEditor
+
+      if (!config.isEnabled || !editor) {
```

Comment:
isEnabled flag used to basically disable all of the functionality, now it only diables highlighting and status bar, but description still says: `"Enable or disable the extension"`, maybe it should explicitly say that it will disable highlighting and status bar, not the whole extension (for that, you just disable it).

Reply:
Ok, I will change that